src/boards/include/boards/pico.h

```diff
@@ -81,4 +81,19 @@
 #define PICO_RP2040_B0_SUPPORTED 1
 #endif
 
+// Pin get VBUS
+#ifndef PICO_VBUS_PIN
+#define PICO_VBUS_PIN 24
+#endif
+
+// Pin used to monitor VSYS
+#ifndef PICO_VSYS_PIN
+#define PICO_VSYS_PIN 29
+#endif
+
+// ADC to use to monitor VSYS
+#ifndef PICO_VSYS_ADC_INPUT
+#define PICO_VSYS_ADC_INPUT 3
+#endif
```

Comment:
Isn't there a 1:1 mapping between GPIO pins and ADC inputs? In which case perhaps having separate defines for both of them is a bit overkill, and it'd be worth having one value calculated from the other one instead?

Reply:
Yes, there's a one to one mapping on Pico boards. I don't think that has to be the case on other boards though. I'm not sure we infer the adc from the pin elsewhere, but I'm prepared to be corrected.

Reply:
IIRC this is actually a chip-level (RP2040) thing rather than a board-level (Pico) thing?